MovieApp/MovieApp/Model/StorageManager.swift

```diff
@@ -62,18 +62,34 @@ class StorageManager {
         }
     }
 
-    func networkStatus(){
+    func getData(at index: String, movieID: Int? = nil){
```

Comment:
Divide this function into 2 separate methods to be called by their respective ViewController